services/ui-src/src/forms/mcpar/dgbs/dgbs.json

```diff
@@ -0,0 +1,113 @@
+{
+  "path": "/mcpar/plan-level-indicators/appeals-state-fair-hearings-grievances/grievances-by-service",
+  "intro": {
+    "section": "Section D: Plan-Level Indicators",
+    "subsection": "Topic IV. Appeals, State Fair Hearings & Grievances",
+    "spreadsheet": {
+      "title": "Find in the Excel Workbook",
+      "descriptionList": ["D1_Plan_Set"]
+    },
+    "info": [
+      {
+        "type": "html",
+        "content": "<h2><strong>Grievances by Service</strong></h2>"
+      },
+      {
+        "type": "p",
+        "content": "Report the number of grievances resolved by plan during the reporting period by service."
+      }
+    ]
+  },
+  "drawer": {
+    "dashboard": {
+      "title": "Report on grievances by service for each plan",
+      "entityType": "plans"
+    },
+    "drawerTitle": "Report encounter data for"
+  },
+  "form": {
+    "id": "dgbs",
+    "fields": [
+      {
+        "id": "dgbs-1",
+        "type": "number",
+        "props": {
+          "label": "D1.IV.15a Resolved grievances related to general inpatient services",
+          "hint": "Enter the total number of grievances resolved by the plan during the reporting year that were related to general inpatient care, including diagnostic and laboratory services. Do not include grievances related to inpatient behavioral health services — those should be included in indicator D1.IV.15c. If the managed care plan does not cover this type of service, enter \"N/A\"."
```

Comment:
Each of these fields should have a number mask added because they are values that will need comma-separation.
```suggestion
          "label": "D1.IV.15a Resolved grievances related to general inpatient services",
          "hint": "Enter the total number of grievances resolved by the plan during the reporting year that were related to general inpatient care, including diagnostic and laboratory services. Do not include grievances related to inpatient behavioral health services — those should be included in indicator D1.IV.15c. If the managed care plan does not cover this type of service, enter \"N/A\".",
          "mask": "comma-separated"
```